src/analysis/retail/evoker/augmentation/modules/features/BuffTargetHelper/BuffTargetHelper.tsx

```diff
@@ -443,12 +499,12 @@ class BuffTargetHelper extends Analyzer {
               <a href="https://www.curseforge.com/wow/addons/method-raid-tools">
                 Method Raid Tools
               </a>
```

Comment:
This is missing a spacing
```suggestion
              </a>{' '}
```